internal/sysaccess/sshmgr.go

```diff
@@ -136,6 +141,11 @@ func (s *SSHManager) UpdateKeys(keys []*SSHKey) (retErr error) {
 		}
 		log.Debug("updating %d keys for %s", len(keys), username)
 		if err := s.updateAuthorizedKeysFile(username, keys); err != nil {
+			// TODO: maybe we should always proceed regardless the returned error
```

Comment:
Can we resolve this TODO? In my experience TODO's never get done.

Reply:
On it